packages/x-tailwindcss/src/x-tailwind-plugin/components/icon/default.ts

```diff
@@ -0,0 +1,24 @@
+import { TailwindHelpers } from '../../../types';
+import { iconSizes } from './sizes';
+
+/**
+ * Returns the default styles for component `icon`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function iconDefault(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    fontFamily: theme('fontFamily.icon'),
+    width: 'auto',
+    aspectRatio: '1/1',
+    vectorEffect: 'non-scaling-stroke',
+    flex: '0 0 auto',
```

Comment:
Do we have to add color:currentColor? 

![image](https://user-images.githubusercontent.com/6597815/201028600-d089c8e4-9745-4c02-98e5-bc351ca02fa8.png)

Reply:
It is already added by our tailwind plugin: 
<img width="688" alt="image" src="https://user-images.githubusercontent.com/5759712/201033354-29e09e7a-3256-4e74-b759-283493afc7a3.png">